test/automation/src/sql/notebook.ts

```diff
@@ -132,6 +136,17 @@ export class Notebook {
 		await this.code.waitForElement(expectedResultSelector);
 	}
 
+	async isCodeCellInEditMode(): Promise<boolean> {
+		// verify that cell is in editmode
+		const element = await this.code.waitForElement(`.notebook-cell.active .monaco-editor`);
+		return element.className.includes('focused');
+	}
+
+	async isCodeCellActive(): Promise<void> {
+		// code cell is the active cell
+		await this.code.waitForElement(`.notebook-cell.active .monaco-editor`);
```

Comment:
By convention is statements should always return a boolean. I would instead call this `waitForActiveCodeCell` or even `getActiveCodeCell` and then have it return the element so it can be used in `isCodeCellInEditMode`